tox.ini

```diff
@@ -38,23 +38,25 @@ description =
     devdeps: with the latest developer version of key dependencies
     oldestdeps: with the oldest supported version of key dependencies
     cov: and test coverage
-    numpy116: with numpy 1.16.*
-    numpy117: with numpy 1.17.*
     numpy118: with numpy 1.18.*
-    astropy30: with astropy 3.0.*
+    numpy119: with numpy 1.19.*
+    numpy120: with numpy 1.20.*
+    numpy121: with numpy 1.21.*
     astropy40: with astropy 4.0.*
+    astropy50: with astropy 5.0.*
     astropylts: with the latest astropy LTS
 
 # The following provides some specific pinnings for key packages
 deps =
 
-    numpy116: numpy==1.16.*
-    numpy117: numpy==1.17.*
     numpy118: numpy==1.18.*
+    numpy119: numpy==1.19.*
+    numpy120: numpy==1.20.*
+    numpy121: numpy==1.21.*
 
-    astropy30: astropy==3.0.*
     astropy40: astropy==4.0.*
-    astropylts: astropy==4.0.*
+    astropy50: astropy==5.0.*
+    astropylts: astropy==5.0.*
```

Comment:
on ln 47: astropalts: with the latest astropy LTS while here it is satrapy 5.0, ok?

Reply:
The latest LTS version is 5.0.

Reply:
Added this in the comment on line 47.